pnpm-lock.yaml

```diff
@@ -3254,11 +3254,485 @@ importers:
         specifier: ^5
         version: 5.0.4
 
+  packages/libuild/libuild-core:
```

Comment:
The lock file size has increased by about 3000+ lines, we should optimize it in the future.

Reply:
Approve it